bindings/matrix-sdk-ffi/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 // TODO: target-os conditional would be good.
 
-#![allow(unused_qualifications, clippy::new_without_default)]
+#![allow(unused_qualifications, clippy::new_without_default, unused_macros)]
```

Comment:
Why is this needed?

Reply:
The macro isn't used anymore. We can remove it, but it's something Jonas added long ago and I wasn't sure if it could be useful in the future.

Reply:
https://github.com/matrix-org/matrix-rust-sdk/pull/3942/commits/7711a85e04bf598330587bc21b6379aeadc7c0cb